### PR TITLE
migration of filter translations

### PR DESCRIPTION
**Pull request by Mathieu-Deharbe**
Migrated from : https://github.com/gridsuite/gridexplore-app/pull/497

### CODE REVIEW

**Comment by basseche** [comment]
Est ce que je peux avoir le droit pour être reviewer sur cette fiche

**Review by basseche — APPROVED** [review]
It s ok but one translation can be approved

File: src/components/translations/description-en.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const descriptionEn = {
+    description: 'Description',
+    descriptionProperty: 'Description (optional)',
+    descriptionModificationError: 'An error when modifying the description',
```
[the remark this answers is not in the document]
Comment (Mathieu-Deharbe) [reply src/components/translations/description-en.ts]: Agreed. But I think "An error occurred while editing the description" is even better.

File: src/components/translations/csv-en.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
```
Comment (ghazwarhili) [inline src/components/translations/csv-en.ts:2]: copyright 2024

File: src/components/translations/csv-fr.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
```
Comment (ghazwarhili) [inline src/components/translations/csv-fr.ts:2]: the same here

**Review by ghazwarhili — COMMENTED** [review]
you have to resolve sonarCloud issues (duplicated code)

**Comment by Mathieu-Deharbe** [comment]
> you have to resolve sonarCloud issues (duplicated code)

I don't think I can. This seems to be a sonar limitation : he doesn't analyse strings and therefore he think that those are code duplications. I asked Slimane if there is a way to ignore translation files.

**Pull request merged**